src/js/components/DropButton/index.d.ts

```diff
@@ -2,11 +2,6 @@ import * as React from "react";
 import { ButtonProps } from "../Button";
 
 export interface DropButtonProps {
-  a11yTitle?: string;
```

Comment:
Why are these properties being removed from DropButton?

Reply:
I think it is already included in `ButtonProps`, line 13.

Reply:
Ah, yes. I didn't see that in the diff.